tensorboard/plugins/pr_curve/summary.py

```diff
@@ -31,24 +31,19 @@ def op(
     labels,
     predictions,
     num_thresholds=200,
-    weights=None,
-    num_classes=1,
+    weight=1.0,
     display_name=None,
-    description=None):
-  """Computes multi-class PR summaries for a list of thresholds in `[0, 1]`.
+    description=''):
```

Comment:
Woah. This and java reflection probably consist of my top 2 blatant violations of abstraction barriers.